src/Engine/Systems/Subsystems/Physics/OrbitPointGen.hpp

```
@@ -0,0 +1,265 @@
+        const COE::Elements &coe,
+        const glm::dmat3 &R,
+        const GenerationConfig &config)
+    {
+        std::vector<glm::dvec3> points;
+        points.reserve(config.pointCount);
+
+        double e = coe.e;
+        double p = coe.p;
+
+        // Asymptote angle — true anomaly at which r -> infinity
+        double nu_asymptote = std::acos(-1.0 / e);
```

Clang (14.0.6)

[warning] variable 'nu_asymptote' is not initialized

(cppcoreguidelines-init-variables)